D1673/P1673.md

```diff
@@ -4125,56 +4376,154 @@ template<class Triangle,
          class StorageOrder>
 class layout_blas_packed;
 
-// [linalg.scaled.abs], exposition-only function object @_abs-if-needed_@
+// [linalg.helpers], exposition-only helpers
 
-/* see-below */ @_abs-if-needed_@;
+// [linalg.helpers.abs], exposition-only function object @_abs-if-needed_@
+/* @_see-below_@ */ @_abs-if-needed_@; // @_exposition only_@
 
-// [linalg.scaled.conj], exposition-only function object @_conj-if-needed_@
+// [linalg.helpers.conj], exposition-only function object @_conj-if-needed_@
+/* @_see-below_@ */ @_conj-if-needed_@; // @_exposition only_@
 
-/* see-below */ @_conj-if-needed_@;
+// [linalg.helpers.real], exposition-only function object @_real-if-needed_@
+/* @_see-below_@ */ @_real-if-needed_@; // @_exposition only_@
 
-// [linalg.scaled.real], exposition-only function object @_real-if-needed_@
+// [linalg.helpers.imag], exposition-only function object @_imag-if-needed_@
+/* @_see-below_@ */ @_imag-if-needed_@; // @_exposition only_@
 
-/* see-below */ @_real-if-needed_@;
+// [linalg.helpers.concepts], linear algebra argument concepts
+
+template<class T>
+struct @_is-mdspan_@; // @_exposition only_@
 
-// [linalg.scaled.imag], exposition-only function object @_imag-if-needed_@
+template<class T>
+concept @_in-vector_@; = // @_exposition only_@
 
-/* see-below */ @_imag-if-needed_@;
+template<class T>
+concept @_out-vector_@; = // @_exposition only_@
 
-// [linalg.scaled.scaledaccessor], class template scaled_accessor
+template<class T>
+concept @_inout-vector_@; = // @_exposition only_@
 
+template<class T>
+concept @_in-matrix_@; = // @_exposition only_@
+
+template<class T>
+concept @_out-matrix_@; = // @_exposition only_@
+
+template<class T>
+concept @_inout-matrix_@; = // @_exposition only_@
+
+template<class T>
+concept @_possibly-packed-inout-matrix_@; = // @_exposition only_@
+
+template<class T>
+concept @_in-object_@; = // @_exposition only_@
+
+template<class T>
+concept @_out-object_@; = // @_exposition only_@
+
+template<class T>
+concept @_inout-object_@; = // @_exposition only_@
```

Comment:
whats up with these = signes?

Reply:
@crtrott Probalby a typo.  I'll take a look and add the fix to the next PR.

Reply:
@crtrott PR https://github.com/ORNL/cpp-proposals-pub/pull/429 fixes these typos.  Thanks for pointing this out!